lib/git-shell-out-strategy.js

```diff
@@ -80,6 +85,12 @@ export default class GitShellOutStrategy {
         PATH: process.env.PATH || '',
       };
 
+      ENV_VARS_TO_COPY.forEach(envVar => {
+        if (process.env[envVar] !== undefined) {
+          env[envVar] = process.env[envVar];
+        }
+      });
+
```

Comment:
Oh I think the other times are in #846 which isn't :shipit: yet. And which would be run from a process that can't import helpers easily.